aviary/docs/user_guide/aviary_commands.ipynb

```diff
@@ -305,6 +305,48 @@
    "source": [
     "!aviary convert_aero_table -h"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(aviary-PMC-command)=\n",
+    "### aviary convert_prop_table\n",
+    "\n",
+    "The `aviary convert_prop_table` command will convert a legacy formatted (GASP) propeller map into an Aviary formatted propeller map.\n",
+    "\n",
+    "Users must provide the path or name of an input deck, the path to the output file, and the engine format being converted.\n",
```

Comment:
can/should we make the output filename optional with it defaulting to the name of the input file with a new file type? (similar to what we are doing in the test)

Reply:
Done.